### PR TITLE
Update Kibana dashboards and Elastic configuration

### PR DESCRIPTION
**Pull request by orltom**

### CODE REVIEW

File: data/elasticsearch/airlock-waf-template.json [hunk 1]
```diff
@@ -1,123 +1,130 @@
 {
-	"index_patterns": [
-		"airlock-waf-default-*",
-		"airlock-logs-*"
```
Comment (orltom) [inline data/elasticsearch/airlock-waf-template.json:4]: FYI: I have noticed that the new template remove the airlock-logs pattern.

File: .github/test/cypress.json [hunk 2]
```diff
@@ -4,5 +4,5 @@
   "video": false,
   "screenshotOnRunFailure": false,
   "chromeWebSecurity": false,
-  "baseUrl": "https://host.docker.internal"
+  "baseUrl": "https://host.docker.internal/"
```
Comment (orltom) [inline .github/test/cypress.json:7]: Without the trailing slash the Kibana test will fail

**Review by dieti80 — CHANGES_REQUESTED** [review]
The changed files look fine to me. Anyway, discover and dashboards does not work anymore and must be fixed.

File: .github/test/cypress.json (hunk 2 above)
Comment (dieti80) [reply .github/test/cypress.json:7]: There shouldn't be any rewrites required. Kibana is configured with the ENV variables to run under /kibana.
See kibana.yaml
The required ENV variables are: SERVER_BASEPATH and SERVER_REWRITEBASEPATH

File: data/elasticsearch/airlock-waf-template.json (hunk 1 above)
Comment (dieti80) [reply data/elasticsearch/airlock-waf-template.json:4]: Everything looks fine in Kibana. Guess this was never required.

**Review by dieti80 — APPROVED** [review]

**Pull request merged**